### PR TITLE
Use OkHttp's interceptor to measure request time.

### PR DESCRIPTION
**Pull request by zshbleaker**
Which is more accurate.

### CODE REVIEW

**Comment by zshbleaker** [comment]
现在的 duration 非常不准，数字基本在秒级别，远高于正常水平。

File: library/src/main/java/com/qiniu/android/http/Client.java [hunk 1]
```diff
@@ -304,7 +304,8 @@ public void run() {
 
     }
 
-    private static class IpTag {
+    private static class ResponseTag {
         public String ip = null;
+        public Long duration = null;
```
Comment (longbai) [inline library/src/main/java/com/qiniu/android/http/Client.java]: 那这里没必要装箱，直接long 就好了

Comment (zshbleaker) [inline library/src/main/java/com/qiniu/android/http/Client.java]: Updated.

**Comment by longbai** [comment]
LGTM

**Pull request merged**